cmd/nginx/flags.go

```diff
@@ -231,6 +231,7 @@ Takes the form "<host>:port". If not provided, no admission controller is starte
 	}
 
 	nginx.HealthPath = *defHealthzURL
+	nginx.HealthCheckTimeout = *defHealthCheckTimeout
```

Comment:
execute this only if `*defHealthCheckTimeout > 0` and to 
`nginx.HealthCheckTimeout = *defHealthCheckTimeout * time.Second`